src/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -284,13 +283,12 @@ public byte[] ToByteArray()
             byte highByte;
             int nonZeroDwordIndex = 0;
             uint highDword;
-            uint[] bits = _bits;
-            if (bits == null)
+            if (_bits == null)
             {
-                highByte = (byte)((sign < 0) ? 0xff : 0x00);
-                highDword = unchecked((uint)sign);
+                highByte = (byte)((_sign < 0) ? 0xff : 0x00);
+                highDword = unchecked((uint)_sign);
```

Comment:
Since upgrading to Windows 10, I am having trouble with these steps to view the assembly output.  I am going to try installing the Windows 10 SDK to see if that fixes the issue with loading SOS.